data/actionsmap/yunohost.yml

```diff
@@ -302,6 +302,14 @@ domain:
             arguments:
                 domain:
                     help: Target domain
+                -t:
+                    full: --ttl
+                    help: Time To Live (TTL) is second before DNS servers update. Default is 3600 second (i.e. 1 hour).
```

Comment:
typos : Time To Live (TTL) *in* second before DNS servers update. Default is 3600 second*s* (i.e. 1 hour).